lib/api/controllers/authController.js

```diff
@@ -44,4 +44,41 @@ module.exports = function AuthController (kuzzle) {
         return q.reject(new ResponseObject(requestObject, err));
       });
   };
-};
\ No newline at end of file
+
+  /**
+   * Returns the user identified by the given jwt token
+   * @param {RequestObject} requestObject
+   * @param context
+   * @returns {Promise}
+   */
+  this.getCurrentUser = function (requestObject, context) {
+    kuzzle.pluginsManager.trigger('auth:getCurrentUser', requestObject);
+
+    requestObject.data._id = context.token.user._id;
+
+    return kuzzle.repositories.user.load(requestObject.data._id)
+      .then(user => {
+        var response;
+
+        if (!user) {
+          return q(new ResponseObject(requestObject, {found: false}));
+        }
+
+        if (requestObject.data.body.hydrate !== undefined && requestObject.data.body.hydrate === false) {
```

Comment:
On other routes, by default, hydrate is false. `if (!requestObject.data.body.hydrate)` is enough ?